src/styles/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
```

Comment:
If you add to this class `margin: 0 auto;` then the calendar is aligned horizontally in the center.